docker-compose.yml

```diff
@@ -1,22 +1,31 @@
 version: "3"
 services:
   mmgis:
-    build: .
+    build: 
+      context: .
+      target: dev
+    entrypoint: [ "npm", "run", "start" ]
     depends_on:
       - db
     env_file: .env
     ports:
       - 8888:8888
+      - 8889:8889
     restart: on-failure
     volumes:
       - ./Missions:/usr/src/app/Missions
+      - ./scripts:/usr/src/app/scripts
+      - ./src:/usr/src/app/src
+
   db:
     image: postgis/postgis:10-2.5-alpine
     env_file: .env
     ports:
-      - 5432:5432
+      - 5433:5432
     restart: on-failure
     volumes:
       - mmgis-db:/var/lib/postgresql/data
+      - ./Database:/usr/src/app/Database
+      - ./Database/PgSQL_queries/mmgisDB_queries.sql:/docker-entrypoint-initdb.d/mmgisDB_queries.sql
```

Comment:
Again, the MMGIS ./Database directory is obsolete.

Reply:
Ok! But it was good because I was able to run EMIT locally and load the prod configurations.